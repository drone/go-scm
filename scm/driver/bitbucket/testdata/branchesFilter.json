{
  "pagelen": 30,
  "values": [
    {
      "type": "branch",
      "name": "master",
      "links": {
        "commits": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commits\/master"
        },
        "self": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/refs\/branches\/master"
        },
        "html": {
          "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/branch\/master"
        }
      },
      "target": {
        "hash": "a6e5e7d797edf751cbd839d6bd4aef86c941eec9",
        "repository": {
          "links": {
            "self": {
              "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin"
            },
            "html": {
              "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin"
            },
            "avatar": {
              "href": "https:\/\/bytebucket.org\/ravatar\/%7B7dd600e6-0d9c-4801-b967-cb4cc17359ff%7D?ts=default"
            }
          },
          "type": "repository",
          "name": "stash-example-plugin",
          "full_name": "atlassian\/stash-example-plugin",
          "uuid": "{7dd600e6-0d9c-4801-b967-cb4cc17359ff}"
        },
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "comments": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9\/comments"
          },
          "patch": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/patch\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "html": {
            "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/commits\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "diff": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/diff\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "approve": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9\/approve"
          },
          "statuses": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9\/statuses"
          }
        },
        "author": {
          "raw": "Adam Ahmed <aahmed@atlassian.com>",
          "type": "author",
          "user": {
            "username": "aahmed",
            "display_name": "Adam Ahmed",
            "account_id": "557057:74dc5efb-ffe7-49af-b427-6abc299bb3b9",
            "links": {
              "self": {
                "href": "https:\/\/api.bitbucket.org\/2.0\/users\/aahmed"
              },
              "html": {
                "href": "https:\/\/bitbucket.org\/aahmed\/"
              },
              "avatar": {
                "href": "https:\/\/bitbucket.org\/account\/aahmed\/avatar\/32\/"
              }
            },
            "type": "user",
            "uuid": "{3d5de233-98d4-4138-b4af-8678fbb009ad}"
          }
        },
        "parents": [
          {
            "hash": "5be6855032e171280a1acb860d7265c29f40487c",
            "type": "commit",
            "links": {
              "self": {
                "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/5be6855032e171280a1acb860d7265c29f40487c"
              },
              "html": {
                "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/commits\/5be6855032e171280a1acb860d7265c29f40487c"
              }
            }
          }
        ],
        "date": "2015-08-27T03:25:04+00:00",
        "message": "Add Apache 2.0 License\n",
        "type": "commit"
      }
    },
    {
      "type": "branch",
      "name": "master-patch",
      "links": {
        "commits": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commits\/master"
        },
        "self": {
          "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/refs\/branches\/master"
        },
        "html": {
          "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/branch\/master"
        }
      },
      "target": {
        "hash": "a6e5e7d797edf751cbd839d6bd4aef86c941eec8",
        "repository": {
          "links": {
            "self": {
              "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin"
            },
            "html": {
              "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin"
            },
            "avatar": {
              "href": "https:\/\/bytebucket.org\/ravatar\/%7B7dd600e6-0d9c-4801-b967-cb4cc17359ff%7D?ts=default"
            }
          },
          "type": "repository",
          "name": "stash-example-plugin",
          "full_name": "atlassian\/stash-example-plugin",
          "uuid": "{7dd600e6-0d9c-4801-b967-cb4cc17359ff}"
        },
        "links": {
          "self": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "comments": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9\/comments"
          },
          "patch": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/patch\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "html": {
            "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/commits\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "diff": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/diff\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
          },
          "approve": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9\/approve"
          },
          "statuses": {
            "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/a6e5e7d797edf751cbd839d6bd4aef86c941eec9\/statuses"
          }
        },
        "author": {
          "raw": "Adam Ahmed <aahmed@atlassian.com>",
          "type": "author",
          "user": {
            "username": "aahmed",
            "display_name": "Adam Ahmed",
            "account_id": "557057:74dc5efb-ffe7-49af-b427-6abc299bb3b9",
            "links": {
              "self": {
                "href": "https:\/\/api.bitbucket.org\/2.0\/users\/aahmed"
              },
              "html": {
                "href": "https:\/\/bitbucket.org\/aahmed\/"
              },
              "avatar": {
                "href": "https:\/\/bitbucket.org\/account\/aahmed\/avatar\/32\/"
              }
            },
            "type": "user",
            "uuid": "{3d5de233-98d4-4138-b4af-8678fbb009ad}"
          }
        },
        "parents": [
          {
            "hash": "5be6855032e171280a1acb860d7265c29f40487c",
            "type": "commit",
            "links": {
              "self": {
                "href": "https:\/\/api.bitbucket.org\/2.0\/repositories\/atlassian\/stash-example-plugin\/commit\/5be6855032e171280a1acb860d7265c29f40487c"
              },
              "html": {
                "href": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/commits\/5be6855032e171280a1acb860d7265c29f40487c"
              }
            }
          }
        ],
        "date": "2015-08-27T03:25:04+00:00",
        "message": "Add Apache 2.0 License\n",
        "type": "commit"
      }
    }
  ],
  "page": 1,
  "next": "https:\/\/bitbucket.org\/atlassian\/stash-example-plugin\/refs\/branches?pagelen=30&page=2"
}