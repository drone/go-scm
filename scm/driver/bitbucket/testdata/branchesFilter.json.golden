[
    {
        "Name": "master",
        "Path": "refs/heads/master",
        "Sha": "a6e5e7d797edf751cbd839d6bd4aef86c941eec9"
    },
    {
        "Name": "master-patch",
        "Path": "refs/heads/master-patch",
        "Sha": "a6e5e7d797edf751cbd839d6bd4aef86c941eec8"
    }
]